Packages/vcs/Lib/VTKPlots.py

```diff
@@ -328,8 +328,10 @@ def createRenWin(self, *args, **kargs):
             else:
                 width = None
                 height = None
-            width = kargs.get("width", width)
-            height = kargs.get("height", height)
+            if "width" in kargs and kargs["width"] is not None:
+                width = kargs["width"]
+            if "height" in kargs and kargs["height"] is not None:
+                height = kargs["height"]
```

Comment:
Because `width` and `height` are always passed into this function by the [caller](https://github.com/UV-CDAT/uvcdat/blob/8dd1c5fa0796884bb6a6c972b029761e87c21a60/Packages/vcs/Lib/VTKPlots.py#L462-L463), though most of the time they are `None`.